src/Stratis.Bitcoin.Features.Api/ApiFeature.cs

```diff
@@ -119,6 +120,7 @@ public static IFullNodeBuilder UseApi(this IFullNodeBuilder fullNodeBuilder, Act
             {
                 features
                 .AddFeature<ApiFeature>()
+                .DependOn<WalletFeature>()
```

Comment:
The Api shouldn't be dependant on the wallet.

Reply:
So should we do something different with IWalletManager in MiningController?